azure-pipelines.yml

```diff
@@ -159,6 +159,8 @@ jobs:
   steps:
   - template: ci/checkout-template.yml
   - template: ci/install-package-dependencies.yml
+  - bash: ci/scripts/check-bazel-repos.sh
```

Comment:
Nit: The name is a little ambiguous. Maybe emphasize that it's not about repositories, but rather about checking for banned rules. Maybe something like `check-bazel-banned-rules.sh`?

Reply:
Good catch, your right that's confusing